src/Paramore.Brighter.ServiceActivator.Control.Api/Responses/GetStatusResponse.cs

```diff
@@ -44,7 +44,7 @@ public GetStatusResponse(NodeStatus nodeStatus)
     /// <summary>
     /// Timestamp of Status Event
     /// </summary>
-    public DateTime TimeStamp { get; } = DateTime.UtcNow;
+    public DateTimeOffset TimeStamp { get; } = DateTimeOffset.Now;
```

Comment:
Do we want UtcNow for defaults. Does the local time of the computer matter that we want the offset from it?

Reply:
I'm not precious about this one, I was thinking that if we were translating an offset then we could use Server time.

I'll update it